<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CarDialog</class>
 <widget class="QDialog" name="CarDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLineEdit" name="idEdit"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>id</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="modelEdit"/>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>model</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLineEdit" name="yearEdit"/>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>year</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="priceEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLineEdit" name="regNumberEdit"/>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>reg number</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLineEdit" name="vinEdit"/>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>vin number</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLineEdit" name="passengerSeatsEdit"/>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>seats</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLineEdit" name="doorsEdit"/>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>doors</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>price</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="createButton">
     <property name="text">
      <string>create</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="exitButton">
     <property name="text">
      <string>close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
